lib/undefined-terms-v2.js

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) 2023-2024 Digital Bazaar, Inc. All rights reserved.
+ */
+// Use JSON style for context
+/* eslint quotes: ['error', 'double'] */
+/* eslint quote-props: ['error', 'always'] */
+/* eslint-disable max-len */
```

Comment:
What if we add a custom `eslint` config for these files--which could be either in a dedicated folder (to just these JSON in JS files) or named something like `v1.json.js`. We can then have custom eslint rules for that path or file type that toggle all these things: https://eslint.org/docs/v8.x/use/configure/configuration-files#configuration-based-on-glob-patterns